searchcore/src/vespa/searchcore/proton/server/fast_access_doc_subdb.h

```diff
@@ -92,14 +92,15 @@ class FastAccessDocSubDB : public StoreOnlyDocSubDB
     std::shared_ptr<search::attribute::Interlock> _attribute_interlock;
     DocIdLimit           _docIdLimit;
 
-    std::unique_ptr<AttributeCollectionSpec> createAttributeSpec(const AttributesConfig &attrCfg, const AllocStrategy& alloc_strategy, std::optional<SerialNum> serialNum) const;
     AttributeManager::SP getAndResetInitAttributeManager();
     virtual IFlushTargetList getFlushTargetsInternal() override;
     void reconfigureAttributeMetrics(const IAttributeManager &newMgr, const IAttributeManager &oldMgr);
 
     IReprocessingTask::UP createReprocessingTask(IReprocessingInitializer &initializer,
                                                  const std::shared_ptr<const document::DocumentTypeRepo> &docTypeRepo) const;
 
+    bool get_fast_access_attributes_only() const noexcept { return _fastAccessAttributesOnly; }
```

Comment:
Consider rename to _has_fast_access_attributes_only()_.

Reply:
Done.